src/math/libm.cr

```diff
@@ -1,4 +1,4 @@
-{% if flag?(:linux) || flag?(:freebsd) || flag?(:openbsd) || flag?(:dragonfly) %}
+{% unless flag?(:darwin) || flag?(:win32) %}
```

Comment:
I'd rather list the targets for which this code is intended, instead of having to deduce them:
```suggestion
{% if flag?(:linux) || flag?(:bsd) %}
```